state/replay.go

```diff
@@ -153,6 +158,9 @@ func (r *Replay) UnmarshalBinary(p []byte) error {
 }
 
 func (r *Replay) Save() *Replay {
+	// KLUDGE don't make a copy to prevent memory leak
```

Comment:
This caused some test failures - but did 'fix' the mem lean - though there is still a leak that seems to be from SaveDBStateList()

Reply:
likely not what we want...